--- node_modules/@vechain/sdk-network/dist/index.org.js	2024-07-27 17:14:41
+++ node_modules/@vechain/sdk-network/dist/index.js	2024-07-27 17:15:07
@@ -4389,9 +4389,9 @@
      *VeChain supports multiple clauses in one transaction, thus the actual data should be obtained by looking into each clause.
      * Due to the single clause limitation of Ethereum, we assume the first clause is the clause from which we obtain the data.
      */
-    input: tx.clauses[0].data,
-    to: tx.clauses[0].to,
-    value: import_sdk_core24.Quantity.of(tx.clauses[0].value),
+    input: tx.clauses[0]?.data ?? '0x',
+    to: tx.clauses[0]?.to,
+    value: import_sdk_core25.Quantity.of(tx.clauses[0]?.value ?? '0x0'),
     // Unsupported fields
     gasPrice: "0x0",
     type: "0x0",
@@ -4458,7 +4458,7 @@
     gasUsed: import_sdk_core24.Quantity.of(receipt.gasUsed),
     logs,
     status: receipt.reverted ? "0x0" : "0x1",
-    to: transaction.clauses[0].to,
+    to: transaction.clauses[0]?.to,
     transactionHash: receipt.meta.txID,
     transactionIndex: import_sdk_core24.Quantity.of(transactionIndex),
     // Incompatible fields
